tracing-subscriber/src/layer/mod.rs

```diff
@@ -393,6 +393,38 @@
 //! # Ok(()) }
 //! ```
 //!
+//! ## Runtime-configurable Layers
```

Comment:
also, it seems like this section is _maybe_ also a good place to mention using `Reload` for _dynamic_ runtime configuration? but, it would be fine to add that in a separate branch.

Reply:
I'll add a note that `Reload` can be used for dynamic _re_-configuration.